docs/contribute/installer.md

```diff
@@ -49,115 +48,115 @@ The real installer handles all stuff. It supports python2.7 (not tested on earli
+-   Note - It's the same as the WasmEdge version if not specified.
 
 #### Tensorflow Extensions Dependencies Version
 
-* Full Option: `--tf-deps-version TF_DEPS_VERSION`
-* Description: Install the given VERSION of the dependencies of the Tensorflow and Tensorflow lite extension. Only available when the `Extensions` is set to `all` or `tensorflow`.
-* Note - It's the same as the WasmEdge version if not specified.
+-   Full Option: `--tf-deps-version TF_DEPS_VERSION`
+-   Description: Install the given VERSION of the dependencies of the Tensorflow and Tensorflow lite extension. Only available when the `Extensions` is set to `all` or `tensorflow`.
+-   Note - It's the same as the WasmEdge version if not specified.
 
 #### Tensorflow Extensions Tools Version
 
-* Full Option: `--tf-tools-version TF_TOOLS_VERSION`
-* Description: Install the given VERSION of the tools of the Tensorflow and Tensorflow lite extension. Only available when the `Extensions` is set to `all` or `tensorflow`.
-* Note - It's the same as the WasmEdge version if not specified.
+-   Full Option: `--tf-tools-version TF_TOOLS_VERSION`
+-   Description: Install the given VERSION of the tools of the Tensorflow and Tensorflow lite extension. Only available when the `Extensions` is set to `all` or `tensorflow`.
+-   Note - It's the same as the WasmEdge version if not specified.
 
 #### Image Extensions Version
 
-* Full Option: `--image-version IMAGE_VERSION`
-* Description: Install the given VERSION of the Image extension. Only available when the `Extensions` is set to `all` or `image`.
-* Note - It's the same as the WasmEdge version if not specified.
+-   Full Option: `--image-version IMAGE_VERSION`
+-   Description: Install the given VERSION of the Image extension. Only available when the `Extensions` is set to `all` or `image`.
+-   Note - It's the same as the WasmEdge version if not specified.
 
 ### Plugins
 
-* Note - Currently `--plugins` is an experimental option.
+-   Note - Currently `--plugins` is an experimental option.
 
-* Full Option: `--plugins wasi_crypto:0.11.0`
+-   Full Option: `--plugins wasi_crypto:0.11.0`
 
-* Note - The format for this argument is `<plugin_name>:<version_number>`. `<version_number>` is not compulsory. For example `--plugins wasi_crypto` is a valid option.
-* Note - `<plugin_name>` is cases sensitive. Allowed values are stated [here](https://wasmedge.org/book/en/plugin.html) in the `Rust Crate` column. The logic is that the release name should be the same.
-* Note - It's the same as the WasmEdge version if not specified.
+-   Note - The format for this argument is `<plugin_name>:<version_number>`. `<version_number>` is not compulsory. For example `--plugins wasi_crypto` is a valid option.
+-   Note - `<plugin_name>` is cases sensitive. Allowed values are stated [here](https://wasmedge.org/book/en/plugin.html) in the `Rust Crate` column. The logic is that the release name should be the same.
```

Comment:
The link should be the new docs.

Reply:
@alabulei1 Could you please let me know if this is the correct link? https://wasmedge.org/docs/category/wasmedge-plugin-system

Reply:
Yes it is.